src/Frontend/Core.hs

```diff
@@ -401,33 +464,36 @@ form formHandler = getH :<|> postH
 -- | Wrap anything that has 'ToHtml' and wrap it in an HTML body with complete page.
 data Frame body
     = Frame { _frameUser :: User, _frameBody :: body, _frameMessages :: [StatusMessage] }
-    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage] }
+    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage] } -- TODO remove messages
```

Comment:
I think it does make sense:

https://github.com/liqd/aula/pull/648/commits/5a953e2e65ca6fcb11ee497e5e968f43e70b3529

any objections?